ui/stories/section.stories.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { storiesOf } from '../.storybook/facade';
+import Section from '../src/section';
+
+
+storiesOf('Section', module)
+  .add('pie chart', () => (
+    <Section config={{
+      section: {
+        dimensions: {
```

Comment:
Apart from section key, will we have any other top level keys?
If not, pls move dimensions and chartType to top level

Reply:
We will have some other props from story level, it will update when we finalize story define